### PR TITLE
DSD-1175: Replace padding for menuButton

### PR DESCRIPTION
**Pull request by isastettler**
Fixes JIRA ticket xxx

## This PR does the following:

- Replaces the padding in multiSelectMenuButton with py, ps, pe to ensure RTL-friendly styling

## How has this been tested?

- run existing tests
- run storybook locally

## Accessibility concerns or updates

-

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [x] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/theme/components/multiSelectMenuButton.ts [hunk 1]
```diff
@@ -28,7 +28,9 @@ const MultiSelectMenuButton = {
       borderWidth: "1px",
       fontSize: "button.default",
       minHeight: { base: defaultElementSizes.mobileFieldHeight, md: "auto" },
-      padding: "8px 8px 8px 16px",
+      py: "8px",
```
Comment (bigfishdesign13) [inline src/theme/components/multiSelectMenuButton.ts]: The styles on lines 31-33 are not quite right.  Please see the correct styles below.
```
px: "s",
py: "xs",
```

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
The styles need to be adjusted.

File: src/theme/components/multiSelectMenuButton.ts (hunk 1 above)
Comment (isastettler) [reply src/theme/components/multiSelectMenuButton.ts]: Updated

**Review by bigfishdesign13 — APPROVED** [review]
Very nice.

**Pull request merged**